test/ctia/test_helpers/core.clj

```diff
@@ -274,16 +275,18 @@
       all-stores
       get-in-config)))
 
+(s/defschema WithAppOptions
+  (st/optional-keys
+   {:enable-http? s/Bool
```

Comment:
How about changing it to `:disable-http?` so absent == false?

Reply:
I've attempted that, but the behavior is controlled by "ctia.http.enabled" property; I think my delicate dance of negating 'disable-http?' to enable the property looked a bit weird, so I didn't do it.